sql/resource_sql_alerts.go

```diff
@@ -70,15 +72,25 @@ func (a *AlertEntity) fromAPIObject(apiAlert *sql.Alert, s map[string]*schema.Sc
 	a.Parent = apiAlert.Parent
 	a.QueryId = apiAlert.Query.Id
 	a.Rearm = apiAlert.Rearm
+
 	a.Options = &AlertOptions{
 		Column:        apiAlert.Options.Column,
 		Op:            apiAlert.Options.Op,
-		Value:         apiAlert.Options.Value,
 		Muted:         apiAlert.Options.Muted,
 		CustomBody:    apiAlert.Options.CustomBody,
 		CustomSubject: apiAlert.Options.CustomSubject,
 	}
 
+	// value can be a string or a float64 - unfortunately this can't be encoded in OpenAPI yet
+	switch value := apiAlert.Options.Value.(type) {
+	case string:
+		a.Options.Value = value
+	case float64:
+		a.Options.Value = strconv.FormatFloat(value, 'f', 0, 64)
```

Comment:
should we convert back to float when in ` toCreateAlertApiObject` ?

Reply:
How do we know if it is supposed to be a float?

It seems that TF mandates that everything has a type, so we aren't allowed to declare that any type can be provided in the HCL and let the provider will deal with it. If we want to handle this, we will need to expose additional fields on AlertOptions like `NumericValue`/`BooleanValue` to let users choose the type of the value they are setting. For now, let's just handle the string case, since that is all that is supported in TF.

Reply:
I think we need to. Let me look into it

Reply:
The cleanest approach would be to map `value: string | number` to `StringValue: string` and `NumberValue: number` in HCL and make both optional. 

The problem is that this would be a backward incompatible change since we currently have `value: string` in HCL. IF we assume that nobody ever used this because it was always broken then we are good but otherwise, we'd have to keep value, map it to `StringValue` and mark it as deprecated.

Reply:
Do we potentially need to handle boolean values?

Reply:
No, just strings and numbers